### ISSUE TITLE
Table history not working: buttons are disabled

### ISSUE THREAD

**Issue by tapestephan** [Sequel-Ace/Sequel-Ace#775]
- Sequel Ace Version (including build number): 3.1.0 Build 3013
- Sequel Ace Source (App Store/GitHub/Homebrew): App Store
- macOS Version: 10.14.6
- Processor Type (Intel/Apple): Intel
- MySQL Version: 5.5.5
- macOS Localization: FR

**Is Issue Present in [Latest Beta](https://github.com/Sequel-Ace/Sequel-Ace/releases)?** Yes (didn't find a newer version than build 3013)

**Description**
As mentioned here https://github.com/Sequel-Ace/Sequel-Ace/issues/758#issuecomment-757460371 by @aramhovsepyan, table history is not working anymore. It used to work in previous versions, but now buttons are disabled.

**Comment by cnk** [comment]
Anyone know which release broke this? The table history is one of my absolute favorite features - so much so that I would like to downgrade until this is fixed.

**Comment by mfehr94** [comment]
> Anyone know which release broke this? The table history is one of my absolute favorite features - so much so that I would like to downgrade until this is fixed.

3.0.2

**Comment by jamesstout** [comment]
I think I broke this in ca4bbfae35afe83ea1a48c31224366bc5e97b282.

I can revert, but then #590 will be reopened.

**Comment by jamesstout** [comment]
Hmmm, 3.1.0 seems to work ok for me. Buttons not disabled.

You mean these buttons left/right cursors at the bottom of the table content view?

<img width="354" alt="Screenshot 2021-01-13 at 7 02 01 PM" src="https://user-images.githubusercontent.com/1576190/104444150-5c99ab80-55d2-11eb-80f2-a1e134307d6b.png">

**Comment by jamesstout** [comment]
Looks like the `spHistoryControllerInstance` was lost from `Interfaces/Base.lproj/DBView.xib` at sometime. Can't quite figure out the blame on a `xib`.

The patch is:
```patch
From bcca5ce4a82cf3513c12b8bbf5184d7cabed3642 Mon Sep 17 00:00:00 2001
From: James Stout <xxxxxx@xxxx.com>
Date: Wed, 13 Jan 2021 21:41:06 +0800
Subject: [PATCH] restored spHistoryControllerInstance

---
 Interfaces/DBView.xib | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/Interfaces/DBView.xib b/Interfaces/DBView.xib
index bd90e3f4..8645b955 100644
--- a/Interfaces/DBView.xib
+++ b/Interfaces/DBView.xib
@@ -46,6 +46,7 @@
                 <outlet property="queryProgressBar" destination="858" id="860"/>
                 <outlet property="renameDatabaseButton" destination="6997" id="7011"/>
                 <outlet property="renameDatabaseMessageField" destination="6995" id="7041"/>
+                <outlet property="spHistoryControllerInstance" destination="6297" id="6298"/>
                 <outlet property="statusTableAccessoryView" destination="6885" id="6897"/>
                 <outlet property="statusTableCopyChecksum" destination="6901" id="6903"/>
                 <outlet property="statusTableView" destination="6889" id="6898"/>
@@ -4124,6 +4125,9 @@ Gw
                             <segment toolTip="Forward" image="NSGoRightTemplate" width="27" enabled="NO" tag="1"/>
                         </segments>
                     </segmentedCell>
+                    <connections>
+                        <action selector="historyControlClicked:" target="6297" id="6300"/>
+                    </connections>
                 </segmentedControl>
                 <popUpButton toolTip="Select a database to view (⇧⌘D)" verticalHuggingPriority="750" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="3995">
                     <rect key="frame" x="0.0" y="3" width="200" height="25"/>
@@ -4372,6 +4376,7 @@ Gw
                 <outlet property="reloadButton" destination="5176" id="6625"/>
                 <outlet property="removeButton" destination="5177" id="5190"/>
                 <outlet property="ruleFilterController" destination="ki9-Po-bdr" id="5od-0U-9xj"/>
+                <outlet property="spHistoryControllerInstance" destination="6297" id="6316"/>
                 <outlet property="tableContentContainer" destination="vcX-Xr-0cm" id="Xak-w0-y6x"/>
                 <outlet property="tableContentView" destination="36" id="142"/>
                 <outlet property="tableDataInstance" destination="4702" id="4712"/>
@@ -4444,6 +4449,7 @@ Gw
                 <outlet property="separatorTableMenuItem3" destination="7924" id="7953"/>
                 <outlet property="showCreateSyntaxContextMenuItem" destination="6923" id="6929"/>
                 <outlet property="showCreateSyntaxMenuItem" destination="7927" id="7949"/>
+                <outlet property="spHistoryControllerInstance" destination="6297" id="6302"/>
                 <outlet property="tableCollationButton" destination="8310" id="8314"/>
                 <outlet property="tableContentInstance" destination="67" id="143"/>
                 <outlet property="tableDataInstance" destination="4702" id="4707"/>
@@ -4595,6 +4601,12 @@ Gw
             </connections>
         </customObject>
         <customObject id="5814" userLabel="SPDatabaseData" customClass="SPDatabaseData"/>
+        <customObject id="6297" userLabel="SPHistoryController" customClass="SPHistoryController">
+            <connections>
+                <outlet property="historyControl" destination="6293" id="6299"/>
+                <outlet property="theDocument" destination="-2" id="6301"/>
+            </connections>
+        </customObject>
         <customObject id="7073" userLabel="SPIndexesController" customClass="SPIndexesController">
             <connections>
                 <outlet property="addIndexButton" destination="5150" id="7076"/>
-- 
2.30.0
```
Will prepare a PR.

**Comment by jamesstout** [comment]
fixed by #783
